yt/yt/server/http_proxy/config.cpp

```diff
@@ -29,6 +29,26 @@ using namespace NAuth;
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void TProfilingEndpointProviderConfig::Register(TRegistrar registrar)
+{
+    registrar.Parameter("component_type", &TThis::ComponentType);
+    registrar.Parameter("monitoring_port", &TThis::MonitoringPort);
+    registrar.Parameter("shards", &TThis::Shards)
+        .Default({"all"});
+    registrar.Parameter("proxy_death_age", &TThis::ProxyDeathAge)
```

Comment:
We already have this parameters in coordinator config, could we at least use that value as a default?

Reply:
I didn't want to drag weird parameters from bootstrap through `CreateSolomonProxy`. But I am kind of okay with adding the whole coordinator config as an extra parameter to `CreateSolomonProxy` and passing the value on from there. 

But the code is annoying either way.